definitions.yaml

```diff
@@ -29,6 +29,68 @@ definitions:
           datacenter:
             description: Identifier of the datacenter or cloud provider region, e. g. "eu-west-1"
             type: string
+          availability_zones:
+            type: object
+            description: Number of availability zones that the cluster can be spread across.
+            properties:
+              max:
+                description: Maximum number of availability zones in this region.
+                type: number
+              default:
+                description: Default number of availability zones for a cluster.
+                type: number
+      network:
+        description: Information related to networking in the clusters
+        type: object
+        properties:
+          ranges:
+            description: IP ranges that can be used by clusters
+            type: array
+            items:
+              type: object
+              properties:
+                name:
+                  description: Name of the IP range (eg production or default)
+                  type: string
+                pods:
+                  description: IP ranges for pods
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that will be used by pods
+                      type: array
+                      items:
+                        type: string
+                services:
+                  description: IP ranges for services in Kubernetes
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that will be used by services in Kubernetes
+                      type: array
+                      items:
+                        type: string
+                docker:
+                  description: IP ranges for docker
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that will be used by docker on the nodes
+                      type: array
+                      items:
+                        type: string
+                clusters:
+                  description: IP ranges for clusters.
+                  type: object
+                  properties:
+                    cidr_blocks:
+                      description: List of CIDRs that are available to create clusters.
+                      type: array
+                      items:
+                        type: string
+                    cidr_mask:
```

Comment:
btw i left out the public/private mask that we currenlty also configure in each installation because this needs to be calculated based on the amount of availability zones.